### PR TITLE
fixed notaion file for hidden files

### PR DESCRIPTION
**Pull request by danial24**
## What I did
fixed notaion file for hidden files
## How to test
N/A
### Is this testable with jest or storyshots?
N/A
### Are any of the existing tests are failing ?
N/A
### Does this need an update to the documentation?
N/A
### Does it have breaking changes ?
N/A
If your answer is yes to any of these, please make sure to include it in your PR.

@bjavaid @amnajunaid please review the changes and merge them.

### CODE REVIEW

File: packages/core/src/utils/shell/copyAll.ts [hunk 1]
```diff
@@ -37,5 +37,5 @@ export const copyAll = async (src: string, dest: string, prompt: boolean = true,
 		shell.mkdir('-p', dest);
 	}
 
-	shell.cp('-rf', src + '/*', dest);
+	shell.cp('-rf', src + '/.*', dest);
```
Comment (artalat) [inline packages/core/src/utils/shell/copyAll.ts]: its tested?

**Comment by danial24** [comment]
tested the regex with certain file name but not with the local cli

File: packages/core/src/utils/shell/copyAll.ts (hunk 1 above)
Comment (danial24) [reply packages/core/src/utils/shell/copyAll.ts]: tested the regex with certain file name but not with the local cli

Comment (artalat) [reply packages/core/src/utils/shell/copyAll.ts]: can you please do that an confirm. we can then go on to merge this

Comment (danial24) [reply packages/core/src/utils/shell/copyAll.ts]: Tested it.

**Pull request merged**